src/components/Explore/DisplayComponent.js

```diff
@@ -276,8 +274,7 @@ class DisplayComponent extends Component {
         dataField: (index + 2).toString(),
         text: column[1],
         sort: true,
-        formatter: this.colFormatter,
-        headerTitle: () => `${column[2]} , ${column[1]}`
```

Comment:
This is implemented for mouse hover of table headers. Could you please add this back?